### ISSUE TITLE
[Bug]  [zos_job_submit] unable to submit member that contain $

### ISSUE THREAD

**Issue by benjaminthompson1** [ansible-collections/ibm_zos_core#724]
### Is there an existing issue for this?

- [X] There are no existing issues.

### Are the dependencies a supported version?

- [X] The dependencies are supported.

### IBM Z Open Automation Utilities

v1.2.2

### IBM Enterprise Python

v3.11.x

### IBM z/OS Ansible core Version

v1.5.0

### ansible-version

latest

### z/OS version

v2.5

### Ansible module

zos_job_submit

### Bug description

   - name: Submit DFH$ECA to create catalog data set and wait for the job to finish.
     zos_job_submit:
       src: “IBMUSER.GIT.JCL.CNTL(DFH$ECA)”
       location: DATA_SET
       wait: true
       wait_time_s: 60
     register: zos_job_submit_result
   - name: DEBUG(-v) Response from Submit DFH$ECA to create catalog data set and wait for the job to finish.
     debug:
       msg: "{{ zos_job_submit_result }} with verbosity 1"
       verbosity: 1
The $ is a special character? I want this as part of the member name to match the CICS documentation. I have also tried src: “IBMUSER.GIT.JCL.CNTL(DFH//$ECA)”
[trace.txt](https://github.com/ansible-collections/ibm_zos_core/files/11145327/trace.txt)

Upon conducting a more in-depth analysis, we discovered that the API might require escaping special characters, but // did not work, but the Ansible code identifies the presence of $, #, or @ symbols. 

### Playbook verbosity output.

see above
[trace.txt](https://github.com/ansible-collections/ibm_zos_core/files/11145356/trace.txt)

### Ansible configuration.

```YAML
[ibmsys1@RHEL ansible-playbooks-zos]$ ansible-config view
# config file for ansible -- https://ansible.com/
# ===============================================

# nearly all parameters can be overridden in ansible-playbook
# or with command line flags. ansible will read ANSIBLE_CONFIG,
# ansible.cfg in the current working directory, .ansible.cfg in
# the home directory or /etc/ansible/ansible.cfg, whichever it
# finds first

[defaults]

# some basic default values...

#inventory      = /etc/ansible/hosts
#library        = /usr/share/my_modules/
#module_utils   = /usr/share/my_module_utils/
#remote_tmp     = ~/.ansible/tmp
#local_tmp      = ~/.ansible/tmp
#plugin_filters_cfg = /etc/ansible/plugin_filters.yml
#forks          = 5
#poll_interval  = 15
#sudo_user      = root
#ask_sudo_pass = True
#ask_pass      = True
#transport      = smart
#remote_port    = 22
#module_lang    = C
#module_set_locale = False

:
```

### Contents of the inventory

```YAML
---
zsystems:
  hosts:
    ben_s0w1:
      ansible_host: 10.1.1.2
      ansible_user: ibmuser
      ansible_port: 65522
      ansible_ssh_private_key_file: ~/.ssh/z25c
```

### Contents of `group_vars` or `host_vars`

```YAML
n/a
```

**Comment by ddimatos** [comment]
Thank you for opening an issue @benjaminthompson1 .

@richp405 this is related to #642 and best addresses in parallel this quarter when #642 is fixed. I believe the solution would be the same based on a simple recreate I provided.

When I disabled our parser in `zos_job_submit` it would allow for the PDS(member) with a special char to submit successfully, the code I disabled was below:
```
        # parser = BetterArgParser(arg_defs)
        # parsed_args = parser.parse_args(module.params)
        parsed_args = module.params
```
Disabling the code is not an option but only a proof of concept. When we encounter a special char, the parser should escape it for the user, we can't allow for escapes because we won't know if that is a escape or intended to be used as part of the name. 

Here is my recreate with JCL:
```
#   eg ansible-playbook -i inventory zos_job_submit_create_member.yml --tags "init,memgen,submem"
---
- hosts: zvm
  collections:
    - ibm.ibm_zos_core
  gather_facts: no
  vars:
    job_name_1: "files/MAKEMEM.jcl"
  environment: "{{ environment_vars }}"

  tasks:
    # **************************************************************************
    #  Make member
    # **************************************************************************

    - name: Create a PDS data set if it does not exist
      zos_data_set:
        name: ANSIBLE.MEMBER.TEST
        type: pds
        space_primary: 5
        space_type: M
        record_format: fb
        record_length: 80
      register: result
      tags: init

    - name: Response
      debug:
        var: result
      tags: init

    - name: Submit {{job_name_1}}
      zos_job_submit:
        src: "{{ playbook_dir }}/{{job_name_1}}"
        location: LOCAL
        wait_time_s: 10
      register: result
      tags: memgen

    - name: Response for submit {{job_name_1}}
      debug:
        var: result
      tags: memgen

    - name: Submit {{job_name_1}}
      zos_job_submit:
        src: ANSIBLE.MEMBER.TEST(SUB$CL)
        location: DATA_SET
        wait_time_s: 10
      register: result
      tags: submem

    - name: Response for submit {{job_name_1}}
      debug:
        var: result
      tags: submem
  ```
  
  JCL:
  ```
//MAKEMEMB    JOB (T043JM,JM00,1,0,0,0),'HELLO WORLD - JRM',
//             MSGCLASS=X,MSGLEVEL=1,NOTIFY=&SYSUID
//STEP1 EXEC PGM=IEBUPDTE
//SYSPRINT DD  SYSOUT=*
//SYSUT1   DD DISP=OLD,DSN=ANSIBLE.MEMBER.TEST
//SYSUT2   DD DISP=OLD,DSN=ANSIBLE.MEMBER.TEST
//SYSIN    DD DATA,DLM=ZZ
./ ADD LIST=ALL,NAME=SUB$CL
//HELLO    JOB (T043JM,JM00,1,0,0,0),'HELLO WORLD - JRM',
//             MSGCLASS=X,MSGLEVEL=1,NOTIFY=&SYSUID
//STEP0001 EXEC PGM=IEBGENER
//SYSIN    DD DUMMY
//SYSPRINT DD SYSOUT=*
//SYSUT1   DD *
HELLO, WORLD
/*
//SYSUT2   DD SYSOUT=*
//
ZZ
/*
//
```

Successful result:
```
fatal: [zvm]: FAILED! => {"changed": false, "duration": 0, "job_id": null, "msg": "The JCL has been submitted ANSIBLE.MEMBER.TEST(SUB$CL) and no job id was returned within the allocated time of 10 seconds. Consider using  module zos_job_query to poll for a long running jobs or increasing the value for `wait_times_s`.", "stderr": "Unable to submit Job: {'rc': 8, 'stdout_response': '', 'stderr_response': 'BGYSC4004E Failed to open file ANSIBLE.MEMBER.TEST(SUB) for read.\\n', 'command': 'jsub \"ANSIBLE.MEMBER.TEST(SUB$CL)\"'}", "stderr_lines": ["Unable to submit Job: {'rc': 8, 'stdout_response': '', 'stderr_response': 'BGYSC4004E Failed to open file ANSIBLE.MEMBER.TEST(SUB) for read.\\n', 'command': 'jsub \"ANSIBLE.MEMBER.TEST(SUB$CL)\"'}"]}
```

The error when you try to escape it:
```
fatal: [zvm]: FAILED! => {"changed": false, "msg": "Parameter verification failed", "stderr": "Invalid argument \"ANSIBLE.MEMBER.TEST(SUB\\$CL)\" for type \"data_set\" or \"path\".", "stderr_lines": ["Invalid argument \"ANSIBLE.MEMBER.TEST(SUB\\$CL)\" for type \"data_set\" or \"path\"."]}
```

**Comment by richp405** [comment]
We have validated a code change that will properly escape the characters, but need to wait for the matching component in zoau to handle this change.  Ticket filed is Jira/Narare/10103

**Comment by ddimatos** [comment]
This relies on ZOAU 1.3 , IBM z/OS core collection will support 1.3 in Q1 2024

**Comment by richp405** [comment]
Going to put the symbol-escape login into PR for Friday... it won't do any harm to have it ready, even if we ice it until closer to 1.3 release.

This is for 642 and 724... not sure if this addresses 416, but I believe zoau is working that in a different part of their plan for 1.3

**Comment by ddimatos** [comment]
Based on the discussion we had today, we decided this work needs careful consideration and should not be included in 1.8.0-beta.1 and instead try for 1.9.0-beta.1 which will still be a NOOP until zoua 1.3 but the code code be ready for early build testing. Moving this to Q4.5 since there is capacity for you there and will post plan comment this work as it is out of plan.

**Comment by fernandofloresg** [comment]
Ran this test in the latest dev branch, and can confirm this was fixed by #1375 .

```
(AC2.16PY3.9) fernandoflores@Fernandos-MacBook-Pro zos_job_submit-member-char % ansible-playbook -i inventory.yml 1_submit.yml
[WARNING]: Skipping plugin
(/Users/fernandoflores/Documents/IBM/Repos/ibm_zos_core/plugins/modules/zos_apf.py)
as it seems to be invalid: module 'ansible.plugins.connection.zos_apf' has no
attribute 'Connection'
[WARNING]: Skipping plugin (/Users/fernandoflores/Documents/IBM/Repos/ibm_zos_core/pl
ugins/modules/zos_archive.py) as it seems to be invalid: module
'ansible.plugins.connection.zos_archive' has no attribute 'Connection'
[WARNING]: Skipping plugin (/Users/fernandoflores/Documents/IBM/Repos/ibm_zos_core/pl
ugins/modules/zos_backup_restore.py) as it seems to be invalid: module
'ansible.plugins.connection.zos_backup_restore' has no attribute 'Connection'
[WARNING]: Skipping plugin (/Users/fernandoflores/Documents/IBM/Repos/ibm_zos_core/pl
ugins/modules/zos_blockinfile.py) as it seems to be invalid: module
'ansible.plugins.connection.zos_blockinfile' has no attribute 'Connection'
[WARNING]: Skipping plugin
(/Users/fernandoflores/Documents/IBM/Repos/ibm_zos_core/plugins/modules/zos_copy.py)
as it seems to be invalid: module 'ansible.plugins.connection.zos_copy' has no
attribute 'Connection'
[WARNING]: Skipping plugin (/Users/fernandoflores/Documents/IBM/Repos/ibm_zos_core/pl
ugins/modules/zos_data_set.py) as it seems to be invalid: module
'ansible.plugins.connection.zos_data_set' has no attribute 'Connection'
[WARNING]: Skipping plugin (/Users/fernandoflores/Documents/IBM/Repos/ibm_zos_core/pl
ugins/modules/zos_encode.py) as it seems to be invalid: module
'ansible.plugins.connection.zos_encode' has no attribute 'Connection'
[WARNING]: Skipping plugin
(/Users/fernandoflores/Documents/IBM/Repos/ibm_zos_core/plugins/modules/zos_fetch.py)
as it seems to be invalid: module 'ansible.plugins.connection.zos_fetch' has no
attribute 'Connection'
[WARNING]: Skipping plugin
(/Users/fernandoflores/Documents/IBM/Repos/ibm_zos_core/plugins/modules/zos_find.py)
as it seems to be invalid: module 'ansible.plugins.connection.zos_find' has no
attribute 'Connection'
[WARNING]: Skipping plugin (/Users/fernandoflores/Documents/IBM/Repos/ibm_zos_core/pl
ugins/modules/zos_gather_facts.py) as it seems to be invalid: module
'ansible.plugins.connection.zos_gather_facts' has no attribute 'Connection'
[WARNING]: Skipping plugin (/Users/fernandoflores/Documents/IBM/Repos/ibm_zos_core/pl
ugins/modules/zos_job_output.py) as it seems to be invalid: module
'ansible.plugins.connection.zos_job_output' has no attribute 'Connection'
[WARNING]: Skipping plugin (/Users/fernandoflores/Documents/IBM/Repos/ibm_zos_core/pl
ugins/modules/zos_job_query.py) as it seems to be invalid: module
'ansible.plugins.connection.zos_job_query' has no attribute 'Connection'
[WARNING]: Skipping plugin (/Users/fernandoflores/Documents/IBM/Repos/ibm_zos_core/pl
ugins/modules/zos_job_submit.py) as it seems to be invalid: module
'ansible.plugins.connection.zos_job_submit' has no attribute 'Connection'
[WARNING]: Skipping plugin (/Users/fernandoflores/Documents/IBM/Repos/ibm_zos_core/pl
ugins/modules/zos_lineinfile.py) as it seems to be invalid: module
'ansible.plugins.connection.zos_lineinfile' has no attribute 'Connection'
[WARNING]: Skipping plugin
(/Users/fernandoflores/Documents/IBM/Repos/ibm_zos_core/plugins/modules/zos_mount.py)
as it seems to be invalid: module 'ansible.plugins.connection.zos_mount' has no
attribute 'Connection'
[WARNING]: Skipping plugin (/Users/fernandoflores/Documents/IBM/Repos/ibm_zos_core/pl
ugins/modules/zos_mvs_raw.py) as it seems to be invalid: module
'ansible.plugins.connection.zos_mvs_raw' has no attribute 'Connection'
[WARNING]: Skipping plugin (/Users/fernandoflores/Documents/IBM/Repos/ibm_zos_core/pl
ugins/modules/zos_operator.py) as it seems to be invalid: module
'ansible.plugins.connection.zos_operator' has no attribute 'Connection'
[WARNING]: Skipping plugin (/Users/fernandoflores/Documents/IBM/Repos/ibm_zos_core/pl
ugins/modules/zos_operator_action_query.py) as it seems to be invalid: module
'ansible.plugins.connection.zos_operator_action_query' has no attribute 'Connection'
[WARNING]: Skipping plugin
(/Users/fernandoflores/Documents/IBM/Repos/ibm_zos_core/plugins/modules/zos_ping.py)
as it seems to be invalid: module 'ansible.plugins.connection.zos_ping' has no
attribute 'Connection'
[WARNING]: Skipping plugin (/Users/fernandoflores/Documents/IBM/Repos/ibm_zos_core/pl
ugins/modules/zos_script.py) as it seems to be invalid: module
'ansible.plugins.connection.zos_script' has no attribute 'Connection'
[WARNING]: Skipping plugin (/Users/fernandoflores/Documents/IBM/Repos/ibm_zos_core/pl
ugins/modules/zos_tso_command.py) as it seems to be invalid: module
'ansible.plugins.connection.zos_tso_command' has no attribute 'Connection'
[WARNING]: Skipping plugin (/Users/fernandoflores/Documents/IBM/Repos/ibm_zos_core/pl
ugins/modules/zos_unarchive.py) as it seems to be invalid: module
'ansible.plugins.connection.zos_unarchive' has no attribute 'Connection'
[WARNING]: Skipping plugin (/Users/fernandoflores/Documents/IBM/Repos/ibm_zos_core/pl
ugins/modules/zos_volume_init.py) as it seems to be invalid: module
'ansible.plugins.connection.zos_volume_init' has no attribute 'Connection'
[WARNING]: While constructing a mapping from
/Users/fernandoflores/Documents/IBM/tasks/wip/zos_job_submit-member-
char/1_submit.yml, line 1, column 3, found a duplicate dict key (tasks). Using last
defined value only.

PLAY [zvm] ***************************************************************************

TASK [Create a PDS data set if it does not exist] ************************************
changed: [zvm]

TASK [Response] **********************************************************************
ok: [zvm] => {
    "result": {
        "changed": true,
        "failed": false,
        "message": "",
        "names": [
            "ANSIBLE.MEMBER.TEST"
        ]
    }
}

TASK [Submit files/sample.jcl] *******************************************************
changed: [zvm]

TASK [Response for submit files/sample.jcl] ******************************************
ok: [zvm] => {
    "result": {
        "changed": true,
        "dest": "/tmp/ansible.lhuumas8",
        "duration": 2,
        "failed": false,
        "gid": 1,
        "group": "OMVSGRP",
        "is_binary": false,
        "jobs": [
            {
                "asid": 0,
                "class": "A",
                "content_type": "",
                "creation_date": "2024-06-27",
                "creation_time": "6:46:37",
                "ddnames": [
                    {
                        "byte_count": 133,
                        "content": [
                            "1                       J E S 2  J O B  L O G  --  S Y S T E M  S T L 1  --  N O D E  S T L 1",
                            "0     ",
                            " 16.46.37 JOB00246 ---- THURSDAY,  27 JUN 2024 ----",
                            " 16.46.37 JOB00246  IRR010I  USERID OMVSADM  IS ASSIGNED TO THIS JOB.",
                            " 16.46.37 JOB00246  $HASP375 MAKEMEMB ESTIMATED  LINES EXCEEDED",
                            " 16.46.38 JOB00246  ICH70001I OMVSADM  LAST ACCESS AT 16:46:35 ON THURSDAY, JUNE 27, 2024",
                            " 16.46.38 JOB00246  $HASP375 MAKEMEMB ESTIMATED  LINES EXCEEDED",
                            " 16.46.38 JOB00246  $HASP373 MAKEMEMB STARTED - INIT 8    - CLASS A        - SYS STL1",
                            " 16.46.38 JOB00246  SMF000I  MAKEMEMB    STEP1       IEBUPDTE    0000",
                            " 16.46.38 JOB00246  $HASP395 MAKEMEMB ENDED - RC=0000",
                            "0------ JES2 JOB STATISTICS ------",
                            "-  27 JUN 2024 JOB EXECUTION DATE",
                            "-           21 CARDS READ",
                            "-           72 SYSOUT PRINT RECORDS",
                            "-            0 SYSOUT PUNCH RECORDS",
                            "-            8 SYSOUT SPOOL KBYTES",
                            "-         0.00 MINUTES EXECUTION TIME"
                        ],
                        "ddname": "JESMSGLG",
                        "id": 2,
                        "procstep": "-",
                        "record_count": 17,
                        "stepname": "JES2"
                    },
                    {
                        "byte_count": 136,
                        "content": [
                            "        1 //MAKEMEMB    JOB (T043JM,JM00,1,0,0,0),'HELLO WORLD - JRM',            JOB00246",
                            "          //             MSGCLASS=X,MSGLEVEL=1,NOTIFY=&SYSUID",
                            "          IEFC653I SUBSTITUTION JCL - (T043JM,JM00,1,0,0,0),'HELLO WORLD - JRM',MSGCLASS=X,MSGLEVEL=1,NOTIFY=OMVSADM",
                            "        2 //STEP1 EXEC PGM=IEBUPDTE",
                            "        3 //SYSPRINT DD  SYSOUT=*",
                            "        4 //SYSUT1   DD DISP=OLD,DSN=ANSIBLE.MEMBER.TEST",
                            "        5 //SYSUT2   DD DISP=OLD,DSN=ANSIBLE.MEMBER.TEST",
                            "        6 //SYSIN    DD DATA,DLM=ZZ",
                            "          /*",
                            "        7 //"
                        ],
                        "ddname": "JESJCL",
                        "id": 3,
                        "procstep": "-",
                        "record_count": 10,
                        "stepname": "JES2"
                    },
                    {
                        "byte_count": 137,
                        "content": [
                            " ICH70001I OMVSADM  LAST ACCESS AT 16:46:35 ON THURSDAY, JUNE 27, 2024",
                            " IEF043I Actions taken by SMFLIMxx parmlib policy for MAKEMEMB STEP1",
                            "         Step shared page limit set to 2147483647 by policy - SMFLIM00 0001",
                            " IEFA111I MAKEMEMB IS USING THE FOLLOWING JOB RELATED SETTINGS:",
                            "          SWA=BELOW,TIOT SIZE=32K,DSENQSHR=DISALLOW,GDGBIAS=JOB",
                            " IEF236I ALLOC. FOR MAKEMEMB STEP1",
                            " IEF237I JES2 ALLOCATED TO SYSPRINT",
                            " IEF237I 04AF ALLOCATED TO SYSUT1",
                            " IEF237I 04AF ALLOCATED TO SYSUT2",
                            " IEF237I JES2 ALLOCATED TO SYSIN",
                            " IEF142I MAKEMEMB STEP1 - STEP WAS EXECUTED - COND CODE 0000",
                            " IEF285I   OMVSADM.MAKEMEMB.JOB00246.D0000102.?         SYSOUT",
                            " IEF285I   ANSIBLE.MEMBER.TEST                          KEPT",
                            " IEF285I   VOL SER NOS= SCR03 .",
                            " IEF285I   ANSIBLE.MEMBER.TEST                          KEPT",
                            " IEF285I   VOL SER NOS= SCR03 .",
                            " IEF285I   OMVSADM.MAKEMEMB.JOB00246.D0000101.?         SYSIN",
                            " IEF373I STEP/STEP1   /START 2024179.1646",
                            " IEF032I STEP/STEP1   /STOP  2024179.1646",
                            "         CPU:     0 HR  00 MIN  00.00 SEC    SRB:     0 HR  00 MIN  00.00 SEC",
                            "         VIRT:    88K  SYS:   264K  EXT:        8K  SYS:    12052K",
                            "         ATB- REAL:                  1140K  SLOTS:                     0K",
                            "              VIRT- ALLOC:      14M SHRD:       0M",
                            " IEF375I  JOB/MAKEMEMB/START 2024179.1646",
                            " IEF033I  JOB/MAKEMEMB/STOP  2024179.1646",
                            "         CPU:     0 HR  00 MIN  00.00 SEC    SRB:     0 HR  00 MIN  00.00 SEC"
                        ],
                        "ddname": "JESYSMSG",
                        "id": 4,
                        "procstep": "-",
                        "record_count": 26,
                        "stepname": "JES2"
                    },
                    {
                        "byte_count": 121,
                        "content": [
                            "1       SYSIN                            NEW MASTER                                        IEBUPDTE LOG PAGE 0001",
                            "-       ./ ADD LIST=ALL,NAME=SUB$CL",
                            "                                         //HELLO    JOB (T043JM,JM00,1,0,0,0),'HELLO WORLD - JRM',",
                            "                                         //             MSGCLASS=X,MSGLEVEL=1,NOTIFY=&SYSUID",
                            "                                         //STEP0001 EXEC PGM=IEBGENER",
                            "                                         //SYSIN    DD DUMMY",
                            "                                         //SYSPRINT DD SYSOUT=*",
                            "                                         //SYSUT1   DD *",
                            "                                         HELLO, WORLD",
                            "                                         /*",
                            "                                         //SYSUT2   DD SYSOUT=*",
                            "                                         //",
                            " IEB817I MEMBER NAME (SUB$CL  ) NOT FOUND IN NM DIRECTORY.  STOWED WITH TTR.",
                            " IEB818I HIGHEST CONDITION CODE WAS 00000000",
                            " IEB819I END OF JOB IEBUPDTE."
                        ],
                        "ddname": "SYSPRINT",
                        "id": 102,
                        "procstep": "-",
                        "record_count": 15,
                        "stepname": "STEP1"
                    }
                ],
                "duration": 2,
                "job_class": "A",
                "job_id": "JOB00246",
                "job_name": "MAKEMEMB",
                "owner": "OMVSADM",
                "priority": 1,
                "program_name": "IEBUPDTE",
                "queue_position": 69,
                "ret_code": {
                    "code": 0,
                    "msg": "CC",
                    "msg_code": "0000",
                    "msg_txt": "CC",
                    "steps": [
                        {
                            "step_cc": 0,
                            "step_name": "STEP1"
                        }
                    ]
                },
                "subsystem": "STL1",
                "svc_class": null,
                "system": "STL1"
            }
        ],
        "mode": "0600",
        "owner": "BPXROOT",
        "size": 559,
        "src": "/.ansible/tmp/ansible-tmp-1719532234.0609238-32860-84924150417120/source",
        "state": "file",
        "uid": 0
    }
}

TASK [Submit files/sample.jcl] *******************************************************
changed: [zvm]

TASK [Response for submit files/sample.jcl] ******************************************
ok: [zvm] => {
    "result": {
        "changed": true,
        "duration": 1,
        "failed": false,
        "jobs": [
            {
                "asid": 0,
                "class": "A",
                "content_type": "",
                "creation_date": "2024-06-27",
                "creation_time": "6:46:45",
                "ddnames": [
                    {
                        "byte_count": 133,
                        "content": [
                            "1                       J E S 2  J O B  L O G  --  S Y S T E M  S T L 1  --  N O D E  S T L 1",
                            "0     ",
                            " 16.46.45 JOB00247 ---- THURSDAY,  27 JUN 2024 ----",
                            " 16.46.45 JOB00247  IRR010I  USERID OMVSADM  IS ASSIGNED TO THIS JOB.",
                            " 16.46.45 JOB00247  $HASP375 HELLO    ESTIMATED  LINES EXCEEDED",
                            " 16.46.45 JOB00247  ICH70001I OMVSADM  LAST ACCESS AT 16:46:44 ON THURSDAY, JUNE 27, 2024",
                            " 16.46.45 JOB00247  $HASP375 HELLO    ESTIMATED  LINES EXCEEDED",
                            " 16.46.45 JOB00247  $HASP373 HELLO    STARTED - INIT 8    - CLASS A        - SYS STL1",
                            " 16.46.45 JOB00247  SMF000I  HELLO       STEP0001    IEBGENER    0000",
                            " 16.46.45 JOB00247  $HASP395 HELLO    ENDED - RC=0000",
                            "0------ JES2 JOB STATISTICS ------",
                            "-  27 JUN 2024 JOB EXECUTION DATE",
                            "-           10 CARDS READ",
                            "-           58 SYSOUT PRINT RECORDS",
                            "-            0 SYSOUT PUNCH RECORDS",
                            "-            7 SYSOUT SPOOL KBYTES",
                            "-         0.00 MINUTES EXECUTION TIME"
                        ],
                        "ddname": "JESMSGLG",
                        "id": 2,
                        "procstep": "-",
                        "record_count": 17,
                        "stepname": "JES2"
                    },
                    {
                        "byte_count": 136,
                        "content": [
                            "        1 //HELLO    JOB (T043JM,JM00,1,0,0,0),'HELLO WORLD - JRM',               JOB00247",
                            "          //             MSGCLASS=X,MSGLEVEL=1,NOTIFY=&SYSUID",
                            "          IEFC653I SUBSTITUTION JCL - (T043JM,JM00,1,0,0,0),'HELLO WORLD - JRM',MSGCLASS=X,MSGLEVEL=1,NOTIFY=OMVSADM",
                            "        2 //STEP0001 EXEC PGM=IEBGENER",
                            "        3 //SYSIN    DD DUMMY",
                            "        4 //SYSPRINT DD SYSOUT=*",
                            "        5 //SYSUT1   DD *",
                            "        6 //SYSUT2   DD SYSOUT=*",
                            "        7 //"
                        ],
                        "ddname": "JESJCL",
                        "id": 3,
                        "procstep": "-",
                        "record_count": 9,
                        "stepname": "JES2"
                    },
                    {
                        "byte_count": 137,
                        "content": [
                            " ICH70001I OMVSADM  LAST ACCESS AT 16:46:44 ON THURSDAY, JUNE 27, 2024",
                            " IEF043I Actions taken by SMFLIMxx parmlib policy for HELLO    STEP0001",
                            "         Step shared page limit set to 2147483647 by policy - SMFLIM00 0001",
                            " IEFA111I HELLO IS USING THE FOLLOWING JOB RELATED SETTINGS:",
                            "          SWA=BELOW,TIOT SIZE=32K,DSENQSHR=DISALLOW,GDGBIAS=JOB",
                            " IEF236I ALLOC. FOR HELLO STEP0001",
                            " IEF237I DMY  ALLOCATED TO SYSIN",
                            " IEF237I JES2 ALLOCATED TO SYSPRINT",
                            " IEF237I JES2 ALLOCATED TO SYSUT1",
                            " IEF237I JES2 ALLOCATED TO SYSUT2",
                            " IEF142I HELLO STEP0001 - STEP WAS EXECUTED - COND CODE 0000",
                            " IEF285I   OMVSADM.HELLO.JOB00247.D0000102.?            SYSOUT",
                            " IEF285I   OMVSADM.HELLO.JOB00247.D0000101.?            SYSIN",
                            " IEF285I   OMVSADM.HELLO.JOB00247.D0000103.?            SYSOUT",
                            " IEF373I STEP/STEP0001/START 2024179.1646",
                            " IEF032I STEP/STEP0001/STOP  2024179.1646",
                            "         CPU:     0 HR  00 MIN  00.00 SEC    SRB:     0 HR  00 MIN  00.00 SEC",
                            "         VIRT:    60K  SYS:   244K  EXT:        0K  SYS:    11996K",
                            "         ATB- REAL:                  1140K  SLOTS:                     0K",
                            "              VIRT- ALLOC:      14M SHRD:       0M",
                            " IEF375I  JOB/HELLO   /START 2024179.1646",
                            " IEF033I  JOB/HELLO   /STOP  2024179.1646",
                            "         CPU:     0 HR  00 MIN  00.00 SEC    SRB:     0 HR  00 MIN  00.00 SEC"
                        ],
                        "ddname": "JESYSMSG",
                        "id": 4,
                        "procstep": "-",
                        "record_count": 23,
                        "stepname": "JES2"
                    },
                    {
                        "byte_count": 121,
                        "content": [
                            "1DATA SET UTILITY - GENERATE                                                                       PAGE 0001",
                            "-IEB352I WARNING: ONE OR MORE OF THE OUTPUT DCB PARMS COPIED FROM INPUT",
                            "      ",
                            " PROCESSING ENDED AT EOD"
                        ],
                        "ddname": "SYSPRINT",
                        "id": 102,
                        "procstep": "-",
                        "record_count": 4,
                        "stepname": "STEP0001"
                    },
                    {
                        "byte_count": 80,
                        "content": [
                            "HELLO, WORLD"
                        ],
                        "ddname": "SYSUT2",
                        "id": 103,
                        "procstep": "-",
                        "record_count": 1,
                        "stepname": "STEP0001"
                    }
                ],
                "duration": 1,
                "job_class": "A",
                "job_id": "JOB00247",
                "job_name": "HELLO",
                "owner": "OMVSADM",
                "priority": 1,
                "program_name": "IEBGENER",
                "queue_position": 70,
                "ret_code": {
                    "code": 0,
                    "msg": "CC",
                    "msg_code": "0000",
                    "msg_txt": "CC",
                    "steps": [
                        {
                            "step_cc": 0,
                            "step_name": "STEP0001"
                        }
                    ]
                },
                "subsystem": "STL1",
                "svc_class": null,
                "system": "STL1"
            }
        ]
    }
}

PLAY RECAP ***************************************************************************
zvm                        : ok=6    changed=3    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
```
